extensions/catalog-crawler/catalog-crawler/src/main/java/de/sovity/edc/ext/catalog/crawler/CrawlerExtension.java

```diff
@@ -90,6 +91,9 @@ public class CrawlerExtension implements ServiceExtension {
     @Inject
     private TypeManager typeManager;
 
+    @Inject
+    private PlaceholderEndpointService placeholderEndpointService;
+
```

Comment:
this has nothing to do with the crawler, it's about the EDC doing a loopback request on itself when sharing an on request data offer's data

Reply:
It's needed because the crawler uses the `HttpDataSourceMapper` which in turn needs the placeholder.
I can put a dummy placeholder if it shouldn't be used.